atm.lua

```diff
@@ -144,16 +143,37 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 
 	trans_ids[p_name] = this_id
 
-	if fields.logout then
-		minetest.show_formspec(p_name, atm_form, main_menu)
-	end
-
-	if fields.new_account then
-		minetest.show_formspec(p_name, atm_form, new_account_fs(p_name))
+	if fields.main then
+		minetest.show_formspec(p_name, atm_form, main_menu(p_name))
 	end
 
-	if fields.info then
-		minetest.show_formspec(p_name, atm_form, info_fs(p_name))
+	if fields.cash then
+		minetest.show_formspec(p_name, atm_form, cash_fs(p_name))
+		local balance = exchange:get_balance(p_name)
+		local inv = minetest.get_inventory({type="detached", name="global_exchange"})
```

Comment:
Is this inventory shared among all users? What if multiple users use this at the same time?